media-libs/gexiv2/gexiv2-0.10.10.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit meson python-r1 toolchain-funcs vala xdg-utils
+
+DESCRIPTION="GObject-based wrapper around the Exiv2 library"
+HOMEPAGE="https://wiki.gnome.org/Projects/gexiv2"
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://gitlab.gnome.org/GNOME/gexiv2.git"
+	inherit git-r3
+else
+	SRC_URI="mirror://gnome/sources/${PN}/$(ver_cut 1-2)/${P}.tar.xz"
```

Comment:
Use gnome.org eclass

Reply:
Would, but gexiv2-0.10.9 uses EAPI7, which is not compatible with that eclass
(because of versionator use). I could patch it to work with newer EAPI, if you'd
like).

Reply:
[Patch](https://archives.gentoo.org/gentoo-dev/message/dc3875525b4138ea3e1316b2e9e696fa) sent to the ml; its pretty cut and dry. This gets merged, and I could use
gnome.org with EAPI 7.

Reply:
Good point about EAPI; just leave it be via SRC_URI then. gnome.org should go together with gnome2-utils and co EAPI-7. Can you add the gnome.org stuff to your pending xdg work (I think it was you..)?

Reply:
Yeah, was me. #9059 Sure, could do.